src/exception_handler/exceptions.py

```diff
@@ -0,0 +1,59 @@
+"""
+This module will contain the various exceptions.
+"""
+
+import logging
+import werkzeug.exceptions as e
+
+class NotFoundException(RuntimeError):
+    """Not found."""
+
+class MyDataNotFound(NotFoundException):
```

Comment:
We dont require MyDataNotFound, in-case if we want to throw not found, the above type should be sufficient.